src/index.html

```diff
@@ -17,6 +26,83 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="https://www.moyo.ua/ua/">
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+          class="logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav">
+          <li class="nav__item">
+            <a
+              href="https://www.moyo.ua/ua/portal_brand/apple_1/"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="https://www.moyo.ua/ua/telecommunication/smart/samsung/"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="https://www.moyo.ua/ua/comp-and-periphery/"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="https://www.moyo.ua/ua/comp-and-periphery/"
+              class="nav__link"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="https://www.moyo.ua/ua/gadgets/"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="https://www.moyo.ua/ua/tablet_el_knigi/tablet/"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="https://www.moyo.ua/ua/foto_video/photo_video/"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="https://www.moyo.ua/ua/foto_video/photo_video/"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
```

Comment:
Remember about correct indentation between parent and child elements. The <li> elements should be indented with 2 spaces inside the <ul> element.